pwnlib/context/__init__.py

```diff
@@ -931,20 +931,22 @@ def log_file(self, value):
         Examples:
 
 
-            >>> context.log_file = 'foo.txt' #doctest: +ELLIPSIS
+            >>> foo_txt = tempfile.mktemp()
+            >>> bar_txt = tempfile.mktemp()
+            >>> context.log_file = foo_txt #doctest: +ELLIPSIS
```

Comment:
Why ELLIPSIS here? I see that it was here before, but it should be enough without it, esp that it does not print a thing
```suggestion
            >>> context.log_file = foo_txt
```

Reply:
I think I just did find-replace and didn't look at the doctest: note.  Will fix.